components/select/src/multi-select/input.js

```diff
@@ -73,6 +73,9 @@ const Input = ({
                 .root-input {
                     overflow-y: auto;
                     flex: 1;
+                    display: flex;
+                    flex-flow: wrap;
+                    row-gap: 4px;
```

Comment:
In https://github.com/dhis2/ui/pull/1224 I added `gap` instead of `row-gap` and I added `marginRight={0}` to the `Chip`.
https://github.com/dhis2/ui/blob/3c44ff68e93b3d4a382ea57e0d49863bbd8e0a22/components/select/src/multi-select/selection-list.js#L46

I am guessing you chose this approach because it is slightly less intrusive. And I guess I'm on board with it. Both solutions would have worked for me.

Reply:
> I am guessing you chose this approach because it is slightly less intrusive.

Yeah, that's right. Not sure what's cleaner though.